tests/accept_manual_start_test.go

```diff
@@ -0,0 +1,150 @@
+// +build apitests
+
+/*
+	Copyright NetFoundry, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package tests
+
+import (
+	"github.com/michaelquigley/pfxlog"
+	"github.com/openziti/fabric/controller/xt_ha"
+	"github.com/openziti/sdk-golang/ziti"
+	"github.com/openziti/sdk-golang/ziti/edge"
+	"github.com/pkg/errors"
+	"testing"
+	"time"
+)
+
+func Test_ManualStart(t *testing.T) {
```

Comment:
I think we are to the point where we have four levels of tests:

1) unit tests
2) internal integration tests
3) API integration tests
4) Connection oriented integration tests

1 and 2 execute together as test files next to the source code.

3 and 4 execute as `apitests`. I feel like they should be split. I am not asking this to be done here, but it is a thought I've had as the category of 4 gets larger.

Reply:
seems like a reasonable distinction. Maybe apitests and dataflow as tags.